### DISCUSSION TITLE
Understanding and thinking about the differences in expression of PassiveDiffusionPl2Int and PassiveDiffusionInt2Cell formulas

### DISCUSSION THREAD

**Discussion by SCZ1174** [Open-Systems-Pharmacology/Forum#1817] [Questions & Problems]
Hi everyone, 
I found PassiveDiffusionPl2Int (Pl2Int) and PassiveDiffusionInt2Cell (Int2Cell) formulas expressed differently in mobi, and I've done a deep dive on them and generated some questions and ideas, I'm not sure if I'm on the right track.
PassiveDiffusionPl2Int = fu * P_endothelial * SA * (C_pls - C_int / K_int_pls)
![image](https://github.com/user-attachments/assets/8c3dba97-9bc6-4594-96fb-059a6b97f84c)
PassiveDiffusionInt2Cell = K_water_int * PA_int_cell * C_int - K_water_cell * PA_cell_int * C_cell
![image](https://github.com/user-attachments/assets/71986b71-6d0e-4dea-828d-1e387a77ae86)

**Question 1:** In Int2Cell, the reason for the separate representation of PA_int_cell and PA_cell_int takes into account the possible presence of transporter proteins in the cell membrane? But in Pl2Int, why is it the same P_endothelial * SA representation in both directions? Is there no transport proteins involved in the transvascular endothelium process from plasma to interstitial (although it is thought that most of  the transport proteins are on the cell membrane)? P_endothelial = 100 cm/min, implying instantaneous equilibration between plasma and interstitial space.

**Question 2:** Can PassiveDiffusionPl2Int be written in the following form? This way everyone can understand the difference between the two formulas.
PassiveDiffusionPl2Int = K_water_int * PA_endothelial * C_pls - K_water_int * PA_endothelial * C_int
The derivation process is as follows:

- PassiveDiffusionPl2Int = fu * P_endothelial * SA * (C_pls - C_int / K_int_pls)
- PassiveDiffusionPl2Int = fu / K_int_pls * P_endothelial * SA * C_pls - fu / K_int_pls * P_endothelial * SA * C_int
- K_water_int = fu / K_int_pls
- PA_endothelial = P_endothelial * SA
![image](https://github.com/user-attachments/assets/49943724-cde4-484a-a932-77fbc86d2844)

**Question 3:** Can you help me decipher what this formula means?
K_int_pls = (f_water_int+f_proteins_int_to_pls * (1/fu – f_water_pls)) * fu
![image](https://github.com/user-attachments/assets/407a322e-c58e-43e2-af7b-536dc96978d2)

**Comment by Proen1** [comment]
Hi @SCZ1174, indeed maybe the passive diffusion equations could be written more homogenously but they are both describing the same
permeability x surface area x ( Unbound concentration Ext-Unbound concentration In).  
The permeability in and out in this equation are the same because this  equation is describing passive permeability and not active transport. Active transport , if existent, it is described in a separate one for active transport, this is an example of how the equation look like for active transport from plasma to interstitial ( transporter in the endothelial layer 
![image](https://github.com/user-attachments/assets/b6f6e451-f03d-4a41-8728-e861fe40ca63)
![image](https://github.com/user-attachments/assets/d123cf99-91b2-4a34-bafa-8f667e4efbe4)

 A  good reason why the active transport enters in a separate equation is because active transport rates are not usually multiplied by surface area and do not depend on the gradient of concentrations.  I am not entirely sure  why the permeability through endothelial layer is so high but for many organs ( e.g. liver) this endothelial layer is discontinuous or fenestrated so it should not be a large barrier for unbound molecules . This an exemption for some organs such as the brain [ref DOI:[10.3390/ijms23031477](http://dx.doi.org/10.3390/ijms23031477)]
![image](https://github.com/user-attachments/assets/8568c7a8-9780-4cda-aa66-5aac66a9f658). I am not sure how this differences in fenestration are accounted in the different tissues, maybe someone else knows..

**Reply by SCZ1174** [reply]
Thank you very much for your detailed reply!
As you said, the permeability of in [P (interstitial->intracellular)→P * P_int_cell_factor(1)] and out [P (intracellular->interstitial)→P * P_cell_int_factor(1)] are equivalent , so why distinguish them here? Or, when will they be unequal?
with best,
Chaozhuang

**Comment by Proen1** [comment]
About the last equation, this is to scale the fraction unbound in the interstitial base don the different of proteins between plasma and interstital fluid . The derivation of this equation is; We first want to know the partition just to interstitial:
 Pint=f_water_int+f_protein_int * kprotein

We do not know kprotein directly but I can get it from the plasma fu that we input.
Pplasma=f_water_plasma+f_protein_plasma* kprotein
Pplasma=1/fu
1/fu=f_water_plasma+f_protein_plasma* kprotein <=>
<=>1/fu- f_water_plasma= f_protein_plasma* kprotein<=>
<=> (1/fu- f_water_plasma)/ f_protein_plasma=kprotein
Then if I replace kprotein the first equation:
Pint=f_water_int+f_protein_int * (1/fu- f_water_plasma)/ f_protein_plasma
We aggregate the fraction of protein in plasma and interstitial in the ration of protein in interstitial and plasma, f_protein_int_to_pls= f_protein_int/ f_protein_plasma
Then, Pint=f_water_int+ f_protein_int_to_pls * (1/fu- f_water_plasma)
Since we want to know Pint/Pplasma, then we get
Pint/Pplasma=f_water_int+ f_protein_int_to_pls * (1/fu- f_water_plasma)/Pplasma
Pint/Pplasma=f_water_int+ f_protein_int_to_pls * (1/fu- f_water_plasma)* fu
I did not confirm but I guess the f_protein_int_to_pls depends on if you are setting albumin or other protein binding partner

**Reply by SCZ1174** [reply]
Thank you very much for your detailed derivation and explanation!
I don't understand some of the parameters and formulas very well, e.g. the meaning of kprotein? the meaning of Pplasma=1/fu?
Is there a better way to systematically understand the parameter and formula settings in Mobi?
with best,
Chaozhuang
